### DISCUSSION TITLE
Menu lateral

### DISCUSSION THREAD

**Discussion by ghost** [frontendbr/forum#245] [Perguntas e Respostas]
Olá galera , 

Como posso fazer um Menu (Lateral) que ao clicar em um ícone ele apareça com fade.. ?

**Comment by jmpas** [comment]
Fala @lucasbicalleto16, esse post do @willianjusten vai te ajudar:
https://willianjusten.com.br/criando-um-menu-off-canvas-com-css3/

**Comment by ghost** [comment]
@Nipher  Blz cara vou dar um olhadinha e te retorno se consegui. :ok:

**Comment by ghost** [comment]
@Nipher Esse está ótimo só falta a `=>` ao invés de `Abrir menu` ,porem eu copio os codigo e não funciona , só o do cara :sa:

**Comment by felipefialho** [comment]
Cara, só acho que você deve entender o funcionamento ao invés de sair copiando o código.

**Comment by jmpas** [comment]
@lucasbicalleto16 é bem o que o @LFeh disse cara, o post explica bem como funciona, vale dar uma lida com calma pra entender direitinho, entendendo você consegue adaptar da forma que você quiser 👍

**Comment by felipefialho** [comment]
Percebo que você tem tendencia a criar várias issues querendo resoluções para suas dúvidas e a maioria das vezes são coisas simples que poderiam ter sido sanadas com uma simples pesquisa no Google. 

Estamos aqui para ajudar. Mas precisamos perceber que está pesquisando e estudando ao invés de estar atrás de soluções prontas.

**Comment by gabrielmanara** [comment]
@lucasbicalleto16 da uma procurada no http://codepen.io/ também. Tem vários exemplos e diferentes implementações pra tudo.

**Comment by afonsopacifer** [comment]
Cheguei a criar um Polymer Element para isso.. Se alguém quiser dar uma olhada :)
https://github.com/afonsopacifer/hamburguer-menu

**Comment by leandroruel** [comment]
acho que o manolo desistiu depois da resposta ali em cima.

**Comment by felipefialho** [comment]
Culpa do @afonsopacifer :smile:

**Comment by afonsopacifer** [comment]
Sei de nada 😄
